src/frontend/src/handler/create_sink.rs

```diff
@@ -230,11 +463,79 @@ pub async fn handle_create_sink(
             ));
 
     let catalog_writer = session.catalog_writer()?;
-    catalog_writer.create_sink(sink.to_proto(), graph).await?;
+    catalog_writer
+        .create_sink(sink.to_proto(), graph, target_table_replace_plan)
+        .await?;
 
     Ok(PgResponse::empty_result(StatementType::CREATE_SINK))
 }
 
+pub(crate) fn insert_merger_to_union(node: &mut StreamNode) {
```

Comment:
```suggestion
pub(crate) fn insert_merger_to_union(node: &mut StreamNode, identity: &str) {
```
so that we can leave the table sink's id and name here?